samvel-aivazian/lesson7-hw3.py

```diff
@@ -0,0 +1,94 @@
+from typing import List
+
+
+class Artist:
+    def __init__(self, name: str, bio: str) -> None:
+        self.name: str = name
+        self.bio: str = bio
+
+
+class Artwork:
+    def __init__(self, title: str, artist: Artist, price: float) -> None:
+        self.title: str = title
+        self.artist: Artist = artist
+        self.price: float = price
+
+    def display_details(self) -> None:
+        print(f"Title: {self.title}, Artist: {self.artist.name}, Price: ${self.price}")
+
+
+class Painting(Artwork):
+    def __init__(self, title: str, artist: Artist, price: float, medium: str) -> None:
+        super().__init__(title, artist, price)
+        self.medium: str = medium
+
+    def display_details(self) -> None:
+        super().display_details()
+        print(f"Medium: {self.medium}")
+
+
+class Sculpture(Artwork):
+    def __init__(self, title: str, artist: Artist, price: float, material: str) -> None:
+        super().__init__(title, artist, price)
+        self.material: str = material
+
+    def display_details(self) -> None:
+        super().display_details()
+        print(f"Material: {self.material}")
```

Comment:
Almost, but I guess it will output `"Medium: {self.medium}"` as it calls `super()`

Is this expected implementation?

Reply:
A medium in the art world is the term for the material used to create artwork.